kernel/src/main/scala/cats/kernel/std/option.scala

```diff
@@ -0,0 +1,68 @@
+package cats.kernel
+package std
+
+package object option extends OptionInstances
+
+trait OptionInstances extends OptionInstances1 {
+  implicit def optionOrder[A: Order] = new OptionOrder[A]
+  implicit def optionMonoid[A: Semigroup] = new OptionMonoid[A]
```

Comment:
I would be open to putting those in the `Monoid` companion object -- I don't think we should put them in `cats.kernel.std` since we wouldn't expose them by default.